src/main/java/duke/Cli.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import java.io.InputStream;
+import java.io.PrintStream;
+import java.util.List;
```

Comment:
maybe can separate them by one line? so the import statements are grouped into groups.